finder/finder.go

```diff
@@ -77,7 +93,8 @@ func (f *Finder) LockDep(dep Dependency) bool {
 	return true
 }
 
-func (f *Finder) FindUrls(dep Dependency) {
+func (f *Finder) FindUrls(dep pom.Dependency) {
+	f.wg.Add(1)
```

Comment:
We've discussed this one, but basically this means that your `wg` isn't exported and also you can't forget to increment your `wg` when using exported functionality that needs to increment your `wg`.